modules/standard/Norm.chpl

```diff
@@ -18,27 +18,42 @@
  */
 
 /*
- *  Matrix and Vector Norms Module
- *
- *  This file defines a module, Norm for computing matrix and
- *  vector norms
- *
- *  The Norm module currently defines:
- *  - 1-norm, 2-norm, infinity norm and Frobenius norm for vectors
- *  - 1-norm, infinity norm and Frobenius norm for matrices
- * 
- *  The normType argument for the first few overloads of norm() is an
- *  enumeration indicating the norm to compute.
- *
- * The norm function may also be called without specifying normType,
- * in which case the default norm for that array rank will be used.
+  Matrix and Vector Norms
+
+  The Norm module currently defines:
+
+    * 1-norm, 2-norm, infinity norm and Frobenius norm for 1D arrays
+    * 1-norm, infinity norm and Frobenius norm for 2D arrays
+
+  The :class:`normType` argument for the first few overloads of :proc:`!norm` is
+  an enumeration indicating the norm to compute.
+
+  The norm function may also be called without specifying the :class:`normType`, in
+  which case the default norm for that array rank will be used.
```

Comment:
It might be useful to say what the defaults are here, too. I don't feel strongly about that, though.